taskcluster/ci/tests/kind.yml

```diff
@@ -41,7 +44,43 @@ tasks:
           make git-modules                                                  &&
 
           echo "Start the dry run"                                          &&
-          make dry-run
+          make dry-run                                                      &&
+          make test-dry-run
+
+  snakemake-run:
+    # Ensure that the snakemake workflow is still executing correctly, even though
+    # taskcluster is the preferred execution environment.
+    worker-type: b-linux-v100-gpu
+    worker:
+      max-run-time: 86400
+      docker-image: {in-tree: train}
```

Comment:
```suggestion

```

Reply:
This `docker-image` attr only applies to `docker-worker` (i.e. a container)

https://github.com/mozilla/firefox-translations-training/blob/main/taskcluster/ci/config.yml#L31

`b-linux-v100-gpu` is a `generic-worker` (i.e. a VM)

Reply:
This `docker-image` attr only applies to `docker-worker` (i.e. a container)

`b-linux-v100-gpu` is a `generic-worker` (i.e. a VM)

https://github.com/mozilla/firefox-translations-training/blob/main/taskcluster/ci/config.yml#L31